### ISSUE TITLE
Broken output for pascal case

### ISSUE THREAD

**Issue by awonihlgard** [iiian/prisma-case-format#40]
Using the example from the readme:
```prisma
model house_rating {
  id       Int    @id @default(autoincrement())
  house_id String
  house    house  @relation(fields: [house_id], references: [id])
  ...
}
...
model house {
  id  String  @id @default(uuid())
  house_rating house_rating[]
  ...
}
```
After running `prisma-case-format --field-case pascal` I get this:
```prisma
Model HouseRating {
@@map("house_rating")
Id       Int @map("id")    @id @default(autoincrement())
HouseId String @map("house_id")
House    House  @relation(fields: [HouseId], references: [Id])
}

Model House {
@@map("house")
Id  String @map("id")  @id @default(uuid())
HouseRating HouseRating[]
}
```

Issues:
- incorrect indendation & formatting
- the `model` keyword has been converted to pascal case `Model`

**Comment by iiian** [comment]
Hi @awonihlgard, thanks for raising this issue. That is supremely hilarious that the demo example for the repo is broken. I haven't had time lately to maintain this project. I can probably commit to getting this fixed around December roughly, if you can bare to wait that long. If you need it sooner, I am happy to suggest where the problem is located, although it's not a terribly complicated repository to begin with.

**Comment by awonihlgard** [comment]
No worries, this doesn't affect my usage. I wanted to use `--map-field-case pascal` (which works) but used `--field-case` instead by mistake, I only created the issue to let you know about the bug :) Your tool saved me a bunch of time so thanks a lot for your work

**Comment by iiian** [comment]
> Your tool saved me a bunch of time so thanks a lot for your work

Glad to hear it!

**Comment by iiian** [comment]
Hey @awonihlgard. This issue should be fixed in `prisma-case-format@2.0.2`.
